src/instruction/incl.rs

```diff
@@ -42,8 +43,12 @@ impl Incl {
         }
     }
 
-    fn fetch_instructions(&self, formatted: &Path) -> Result<Vec<Box<dyn Instruction>>, Error> {
-        let input = std::fs::read_to_string(&formatted)?;
+    fn fetch_instructions(
+        &self,
+        formatted: &Path,
+        reader: &dyn JkReader,
+    ) -> Result<Vec<Box<dyn Instruction>>, Error> {
+        let input = reader.read_to_string(formatted.to_str().unwrap())?;
```

Comment:
Any way to not unwrap here? It might be worth it to add "invalid UTF-8" to the list of errors we should support anyway

Reply:
Fixed by 282dce49e18f7cfb41903a2abce277c4d0df57df